src/main/java/it/pagopa/pn/delivery/svc/AttachmentService.java

```diff
@@ -213,7 +219,7 @@ private void checkAttachmentDigests(NotificationAttachment attachment, String pa
                 .contentType( extractMediaType( fileData.getContentType() ) )
                 .body( new InputStreamResource (fileData.getContent() ) );
 
-        log.debug("AttachmentKey: response {}", response);
+        log.debug("AttachmentKey: response={}", response);
```

Comment:
La response in questo caso è leggibile? Intendo che non è troppo lunga e complessa.